lib/ClangImporter/ImportName.h

```diff
@@ -58,6 +58,8 @@ enum class ImportNameVersion : unsigned {
 };
 enum { NumImportNameVersions = 4 };
 
+static const ImportNameVersion CurrentVersion = ImportNameVersion::Swift3;
+
```

Comment:
@jrose-apple Will this be command-line configurable? That is, should it instead be a function from LangOpts to version? (alternatively, hosted on the Impl)? And, "RequestedVersion" is better name, or something else?